Examples/Sources/MacroExamples/Implementation/Accessor/EnvironmentValueMacro.swift

```diff
@@ -0,0 +1,36 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftSyntaxMacros
+
+public struct EnvironmentValueMacro: AccessorMacro {
+  public static func expansion(
+    of node: AttributeSyntax,
+    providingAccessorsOf declaration: some DeclSyntaxProtocol,
+    in context: some MacroExpansionContext
+  ) throws -> [AccessorDeclSyntax] {
+    guard
+      case let .argumentList(arguments) = node.arguments,
+      let argument = arguments.first
+    else { return [] }
+
+    return [
+      """
+      get { self[\(raw: argument.expression)] }
```

Comment:
I don’t think you need to use `raw: ` here or in the setter.

Reply:
Fixed: https://github.com/apple/swift-syntax/pull/2565/commits/a283c965a0049aeefb68b79e37119451a7178bc2